iit/model_pairs/strict_iit_model_pair.py

```diff
@@ -21,14 +21,9 @@ def __init__(
             training_args: dict = {}
             ):
         default_training_args = {
-            "batch_size": 256,
```

Comment:
Would be really helpful if we could maintain the default args as they were before. Or at least store the default hyperparams we used before in some config for reproducibility.

Reply:
I think all the defaults are preserved (they were just set in multiple places) EXCEPT I did change use_single_loss and optimizer_kwargs. I'll change those back to the defaults from before.